document.go

```diff
@@ -517,13 +520,23 @@ func extractBoolFlag(name string) DocumentFlag {
 	return Invalid
 }
 
+func extractIntFlag(name string) DocumentFlag {
+	if name == "lock_version" {
```

Comment:
Field names like "created_at" and "updated_at" are also not constants, so I've also kept this a literal everywhere 🤷🏻